source/includes/administration/_invoices.md

```diff
@@ -184,3 +184,90 @@ Attributes | &nbsp;
 `detail.categories.products.unit`<br/>*Object* | The unit object of the product.
 `detail.categories.products.unit.unit`<br/>*Object* | The unit value of the product.
 `detail.categories.products.unit.name`<br/>*Object* | The name of the unit of the product in the required language. Only present when defining custom units.
+
+<!-------------------- APPROVE INVOICE -------------------->
+### Approve invoice
+
+`PUT /invoices/:invoice_id/approve`
+
+Manually approve a draft invoice and issue the invoice to the customer by email.
+
+```shell
+# Approve a draft invoice
+curl -X PUT "https://cloudmc_endpoint/rest/invoices/20e9b8d8-b1cb-4462-b6e8-fbb8416b2cbb/approve" \
+   -H "MC-Api-Key: your_api_key"
+```
+> The above command returns a JSON structured like this:
+
+```json
+{
+	"data": {
+		"draftedDate": "2021-07-27T00:00:00Z",
+		"createdDate": "2021-07-27T00:00:00Z",
+		"status": "ISSUED",
+		"organization": {
+			"name": "org_name",
+			"id": "289ec5fb-0970-44e3-bca8-777a691e23c7"
+		},
+		"id": "20e9b8d8-b1cb-4462-b6e8-fbb8416b2cbb",
+		"detail": {
+			"discounted": true,
+			"discounts": [],
+			"startDate": "2021-07-01T00:00:00Z",
+			"endDate": "2021-08-01T00:00:00Z",
+			"currency": "CAD",
+			"subTotal": 272154.29,
+			"categories": [{
+				"discounted": true,
+				"unit": {
+					"unit": "HOUR",
+					"name": {}
+				},
+				"discounts": [],
+				"productId": "5bec5db2-b438-4610-ba73-f3ed50a7b26b",
+				"price": "40.0",
+				"name": {
+					"en": "memory",
+					"fr": "memory"
+				},
+				"subTotal": 65407.39,
+				"sku": "product_sku"
+			}]
+		},
+		"total": 272154.29,
+	}
+}
+```
+
+Attributes | &nbsp;
+---- | -----------
+`id`<br/>*UUID* | The UUID of the invoice.
+`status`<br/>*enum* | The status of the invoice. Possible values are: USAGE_PENDING, DRAFT, ISSUED, OVERDUE, PAID, VOID.
+`createdDate`<br/>*date* | The created date of the invoice.
+`draftedDate`<br/>*date* | The date the invoice was drafted. Is null if status is still USAGE_PENDING.
```

Comment:
We should remove _Is null if status is still USAGE_PENDING._ WDYT?

Reply:
https://github.com/cloudops/cloudmc-api-docs/pull/418/commits/f116010e0dd55c35d39fda4a440e0d90944dd099